includes/polling/os/ekinops-360.inc.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
This should be converted to the new format, see for example https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php#L34

Reply:
I haven't access to Ekinops product anymore. Maybe another contributor can take over for this basic support.